### PR TITLE
more useful messages when module failure

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY
specially if the 'interpreter' is not found

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

cire

##### ANSIBLE VERSION

```
2.x
```

### CODE REVIEW

File: lib/ansible/plugins/action/__init__.py [hunk 1]
```diff
@@ -982,7 +993,7 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         # be sure to remove the BECOME-SUCCESS message now
         out = self._strip_success_message(out)
 
-        display.debug(u"_low_level_execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, out, err))
+        display.debug(u"_low_level_Execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, out, err))
```
Comment (abadger) [inline lib/ansible/plugins/action/__init__.py]: This line looks like it only introduces a typo ;-)

File: lib/ansible/plugins/action/__init__.py [hunk 2]
```diff
@@ -896,12 +899,20 @@ def _parse_returned_data(self, res):
...
             data = dict(failed=True, _ansible_parsed=False)
-            data['msg'] = "MODULE FAILURE"
             data['module_stdout'] = res.get('stdout', u'')
             if 'stderr' in res:
                 data['module_stderr'] = res['stderr']
                 if res['stderr'].startswith(u'Traceback'):
                     data['exception'] = res['stderr']
+
+            # try to figure out if we are missing interpreter
+            if self._used_interpreter is not None and '%s: No such file or directory' % self._used_interpreter.lstrip('!#')in data['module_stderr']:
```
Comment (abadger) [inline lib/ansible/plugins/action/__init__.py]: pep8 will probably complain about missing a space before the ```in```.

**Comment by abadger** [comment]
Noted some typos but otherwise looks good to me.

File: lib/ansible/plugins/action/__init__.py (hunk 1 above)
Comment (bcoca) [reply lib/ansible/plugins/action/__init__.py]: last minute updates ...

**Comment by abadger** [comment]
rebuild_merge

**Pull request merged**